arwen/contexts/validator.go

```diff
@@ -47,6 +47,22 @@ func (validator *wasmValidator) verifyFunctions(instance wasmer.InstanceHandler)
 	return nil
 }
 
+var protectedFunctions = map[string]bool{
+	"internalVMErrors":  true,
+	"transferValueOnly": true}
+
+func (validator *wasmValidator) verifyProtectedFunctions(instance wasmer.InstanceHandler) error {
+	for functionName := range instance.GetExports() {
```

Comment:
this is not backward compatible. please use flag.

Reply:
use flagManagedCryptoAPI - that is the next one which is going to be activated.

Reply:
done